keyboards/redragon/k552/v2/iso/keyboard.json

```diff
@@ -35,20 +35,20 @@
     "rgb_matrix": {
         "animations": {
             "alphas_mods" : true,
-            "band_pinwheel_sat" : true,
-            "band_pinwheel_val" : true,
-            "band_sat" : true,
-            "band_spiral_sat" : true,
-            "band_spiral_val" : true,
-            "band_val" : true,
+            "band_pinwheel_sat" : false,
```

Comment:
please remove the animations from the json instead of setting to `false`

Reply:
Done!